docs/content/en/docs/yaml-crd-ref/config.md

```diff
@@ -3,3 +3,56 @@ title: KeptnConfig
 description: Define configuration values
 weight: 20
 ---
+
+`KeptnConfig` defines configuration values for the Keptn Lifecycle Toolkit.
+Currently, it is used to configure the URL of the OpenTelemetry collector.
```

Comment:
There's a second config option that will be introduced with the next KLT release so I guess it should already be part of this page then: https://main.lifecycle.keptn.sh/docs/crd-ref/options/v1alpha1/#keptnconfigspec

Reply:
Added to synopsis and example, but need to check that I did it correctly.  I think there might be more to say.

Reply:
done